.circleci/local_publish_helpers.sh

```diff
@@ -269,14 +269,32 @@ function setAwsAccountCredentials {
 }
 
 function runE2eTest {
+    _setupCoverage
     FAILED_TEST_REGEX_FILE="./amplify-e2e-reports/amplify-e2e-failed-test.txt"
 
     if [ -f  $FAILED_TEST_REGEX_FILE ]; then
         # read the content of failed tests
         failedTests=$(<$FAILED_TEST_REGEX_FILE)
-        yarn run e2e --forceExit --no-cache --maxWorkers=4 $TEST_SUITE -t "$failedTests"
+        NODE_V8_COVERAGE=$E2E_TEST_COVERAGE_DIR yarn run e2e --forceExit --no-cache --maxWorkers=4 $TEST_SUITE -t "$failedTests"
```

Comment:
Is there a reason to set this variable on the same line as calling "yarn run e2e"? If there is not a functional reason, I think this would be more legible on two lines.

Reply:
Setting it this way makes sure that it only exists as set for the duration of the following call. Since the V8 coverage is incredibly verbose, this minimizes the chances that something else gets run and produces coverage output.

Reply:
that makes sense. thanks for explaining